apps/sseramemes/src/tweetMeme.ts

```diff
@@ -60,6 +60,9 @@ export const tweetMeme = async (message: Message | PartialMessage) => {
   const mediaId = await uploadMeme(message);
 
   const mediaIds = mediaId ? [mediaId] : undefined;
+  
+  message=message.replace(/@\\/mg,'');
+  message=message.replace(/(@[\w]{1,})#[\d]{1,}/mg,'$1');
```

Comment:
can we avoid mutability?

Reply:
sim, basta criar um novo objeto e guardar a string da mensagem tratada dentro dele antes de encaminhar para o retweet

Reply:
PR de [correção do nome da propriedade](https://github.com/sibelius/ccsseraphini/pull/638) com código de exemplo